Lib/test/test_threading.py

```diff
@@ -1062,6 +1062,13 @@ def test_releasing_unacquired_lock(self):
         self.assertRaises(RuntimeError, lock.release)
 
     def test_recursion_limit(self):
+        # TODO: RUSTPYTHON
```

Comment:
@youknowone I couldn't find an `__init__` file for `test`, plus I thought it would be good to keep the change localized (only this function). I've added here and hopefully the tests will pass now, if you think it should placed elsewhere (maybe `__main__`?) let me know.